### PR TITLE
Update branch-deployment-controls.md

### PR DESCRIPTION
**Pull request by nevikashah**

### CODE REVIEW

File: content/pages/configuration/branch-build-controls.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+When connected to your git source, Pages allows you to control which environments and branches you would like to automatically deploy to. By default, Pages will trigger a deployment any time you commit to either your production or preview environment. However, with branch deployment controls, you can configure automatic deployments to suit your preference on a per project basis.
 
 ## Skip builds
 
-Without any configuration required, you can choose to skip a build and deployment on an adhoc basis. By adding the `[CI Skip]`, `[CI-Skip]`, `[Skip CI]`, `[Skip-CI]` or `[CF-Pages-Skip]` flag as a prefix in your commit message, Pages will omit that build/deployment. The prefixes are case insensitive.
+Without any configuration required, you can choose to skip a deployment on an adhoc basis. By adding the `[CI Skip]`, `[CI-Skip]`, `[Skip CI]`, `[Skip-CI]` or `[CF-Pages-Skip]` flag as a prefix in your commit message, and Pages will omit that deployment. The prefixes are case insensitive.
```
Comment (cmsparks) [inline content/pages/configuration/branch-build-controls.md]: If we're splitting up this page and https://github.com/cloudflare/cloudflare-docs/pull/13680 it feels like the `[CI-skip]` prefix feels like it doesn't really neatly fit into either page.

**Review by cmsparks — APPROVED** [review]
Other than that one comment, LGTM

File: content/pages/configuration/branch-build-controls.md [hunk 2]
```diff
@@ -25,42 +19,29 @@ To configure deployment options, go to your Pages project > **Settings** > **Bui
...
-</ol>
+{{<Aside type="note" header="Wildcard syntax">}}
 
-Pages will process the exclude configuration first, then go to the include configuration. If a branch does not match either then it will be skipped.
+A wildcard (`*`) is a character that is used within rules. It can be placed alone to match anything or placed at the start or end of a rule to allow for better control over branch configuration. A wildcard will match zero or more characters.For example, if you wanted to match all branches that started with `fix/` then you would create the rule `fix/*` to match strings like `fix/1`, `fix/bugs`or `fix/`.
 
-### Wildcards
-
-A wildcard (`*`) is a character that is used within rules. It can be placed alone to match anything or placed at the start or end of a rule to allow for better control over branch configuration. A wildcard will match zero or more characters.
+{{</Aside>}}
```
Comment (irvinebroque) [inline content/pages/configuration/branch-build-controls.md:41]: This explainer feels relevant here? https://github.com/cloudflare/cloudflare-docs/pull/13680/files#diff-6d5f711cfffad80cb71dec04b79ddc7d7d05e0713cda5d1cdb1802d77cb03941R19

**Comment by cmsparks** [comment]
cherry picked the changes from this PR into https://github.com/cloudflare/cloudflare-docs/pull/13693/commits

**Pull request closed without merging**